packages/ember-htmlbars/lib/keywords/get.js

```diff
@@ -0,0 +1,79 @@
+import Stream from "ember-metal/streams/stream";
+import isNone from 'ember-metal/is_none';
+import { labelFor } from "ember-metal/streams/utils";
+import { read, isStream } from "ember-metal/streams/utils";
+import create from "ember-metal/platform/create";
+import merge from "ember-metal/merge";
+import SafeString from "htmlbars-util/safe-string";
+
+export default function getKeyword(morph, env, scope, params, hash, template, inverse, visitor) {
+  var objParam = params[0];
+  var keyParam = params[1];
+
+  Ember.assert("The first argument to {{get}} must be a stream", isStream(objParam));
+  Ember.assert("{{get}} requires at least two arguments", params.length > 1);
+
+  var getStream = new GetStream(objParam, keyParam);
+
+  if (morph === null) {
+    return getStream;
+  } else {
+    env.hooks.inline(morph, env, scope, '-get', [getStream], params);
```

Comment:
You should pass everything through:

`env.hooks.inline(morph, env, scope, '-get', [getStream], params, hash, visitor);`